src/server/server_state.cc

```diff
@@ -67,6 +67,7 @@ void ServerState::Init(uint32_t thread_index, acl::UserRegistry* registry) {
   state_->gstate_ = GlobalState::ACTIVE;
   state_->thread_index_ = thread_index;
   state_->user_registry = registry;
+  state_->shard_slow_log_ = SlowLogShard();
```

Comment:
you do not need this line, each variable in c++ initialized automatically.

Reply:
fixed in https://github.com/dragonflydb/dragonfly/pull/1956/commits/af9e983d02ce2816863fecad8f3abe8dd360212e